src/main/webapp/components/Wizard.js

```diff
@@ -35,6 +35,7 @@ export const Wizard = hh(class Wizard extends Component {
   nextStep = (e) => {
     e.preventDefault();
     if (this.props.isValid(this.state.currentStepIndex, null)) {
+      window.scroll(0, 0);

```

Comment:
nit: this change would be more isolated and less likely to impact different browser configurations if it scrolled to the top of the component instead of going to the top of the window.

Reply:
Hey @rushtong! , how about an Util function, browser compatible scrollToTop() to handle and re-use elsewhere? Does this change covers your request?

Reply:
That would definitely be an improvement, thank you!